components/Tabs/src/MDCTabBar.m

```diff
@@ -426,6 +431,20 @@ - (BOOL)itemBar:(__unused MDCItemBar *)itemBar shouldSelectItem:(UITabBarItem *)
   return shouldSelect;
 }
 
+- (void)itemBar:(MDCItemBar *)itemBar willDisplayItem:(UITabBarItem *)item {
+  id<MDCTabBarDisplayDelegate> delegate = self.displayDelegate;
+  if ([delegate respondsToSelector:@selector(tabBar:willDisplayItem:)]) {
```

Comment:
These are non-optional methods (yay!) so you can invoke the methods safely without checking `respondsToSelector:`